packages/geo-diary-v2/README.md

```diff
@@ -0,0 +1,12 @@
+# geo-diary-v2
+
+geo-diary-v2 for Reapit PAAS platform. Initial scaffold from [React App Boilerplate](https://github.com/reapit/react-app)
+
+## Read on:
+
+- [Getting Started](./src/docs/GETTING_STARTED.md)
```

Comment:
Don't think these docs go anywhere now. We should remove from the scaffold at some point...